CHANGELOG.md

```diff
@@ -15,6 +15,10 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+- [FIXED] Setting a location externally now properly cancels geolocation when still locating
+- [FIXED] Setting a location externally now properly triggers a search
+- [ADDED] Searching for locations that only have geometry data available will now also show a marker at the geometry center.
```

Comment:
This last one, is it a minor or patch update?

Reply:
it's a minor